### PR TITLE
Gt-2322  : Add two languages to forced name

### PR DESCRIPTION
**Pull request by gyasistory**

### CODE REVIEW

File: app/src/main/kotlin/org/cru/godtools/ui/languages/LanguageName.kt [hunk 1]
```diff
@@ -41,28 +41,24 @@ internal fun LanguageName(language: Language, modifier: Modifier = Modifier) {
 
     LanguageName(
         name = remember(context, language, appLanguage) { language.getDisplayName(context, language.code) },
-        secondName = remember(context, language) {
-            language.takeUnless { it.isForcedName }?.getDisplayName(context, appLanguage)
-        },
+        secondName = remember(context, language) { language.getDisplayName(context, appLanguage) },
```
Comment (frett) [inline app/src/main/kotlin/org/cru/godtools/ui/languages/LanguageName.kt:44]: I just noticed that the remember keys are backwards for name and secondName.

`name` only references `context` and `language`, so it doesn't need `appLanguage` as a key.
`secondName` references `context`, `language`, and `appLanguage`, but is currently missing `appLanguage` as a key

**Review by frett — CHANGES_REQUESTED** [review]
noticed an adjacent bug that we should just fix in this PR

File: app/src/main/kotlin/org/cru/godtools/ui/languages/LanguageName.kt (hunk 1 above)
Comment (gyasistory) [reply app/src/main/kotlin/org/cru/godtools/ui/languages/LanguageName.kt:44]: Hi Daniel,

This is the merged in change is this missing the proper support?

https://github.com/dependabot/dependabot-core/pull/7525

**Comment by gyasistory** [comment]
> noticed an adjacent bug that we should just fix in this PR

@frett was this what you commented about?

**Review by frett — APPROVED** [review]

**Pull request merged**